### PR TITLE
fix(dropdown): ensure to not go beyond screen edges

### PR DESCRIPTION
**Pull request by clementprevot**
Add a parameter to avoid going beyond screen edges

Also, fix the detection of mobile touch screen to remove hovering

### CODE REVIEW

File: modules/dropdown/demo/dropdown.html [hunk 1]
```diff
@@ -57,6 +57,12 @@ <h1 class="main-section__title">Dropdown</h1>
                     <td>The dropdown menu open delay in ms (if lx-hover).</td>
                 </tr>
                 <tr>
+                    <td>lx-min-offset</td>
+                    <td><code>integer</code></td>
+                    <td>8px</td>
+                    <td>The offset to keep from the edges of the screen. If the edges of the dropdown should go beyond the edges of the screen, then the dropdown will be bring back to this offset from the edge of the screen.</td>
```
Comment (arnaudvalle) [inline modules/dropdown/demo/dropdown.html]: bring --> brought

Comment (arnaudvalle) [inline modules/dropdown/demo/dropdown.html]: The offset (in px)

File: modules/dropdown/demo/dropdown.html [hunk 2]
```diff
@@ -57,6 +57,12 @@ <h1 class="main-section__title">Dropdown</h1>
                     <td>The dropdown menu open delay in ms (if lx-hover).</td>
                 </tr>
                 <tr>
+                    <td>lx-min-offset</td>
+                    <td><code>integer</code></td>
+                    <td>8px</td>
```
Comment (arnaudvalle) [inline modules/dropdown/demo/dropdown.html]: 8 not 8px I guess?

Comment (clementprevot) [inline modules/dropdown/demo/dropdown.html]: Right

**Pull request merged**